### ISSUE TITLE
The DocVQA train loss can't converge and how many epochs need to be trained?

### ISSUE THREAD

**Issue by dongxuewang-123** [NielsRogge/Transformers-Tutorials#60]
when I use the AdamW optimizer，the loss can't converge:
`model = torch.nn.parallel.DistributedDataParallel(model, device_ids=[args.gpu],find_unused_parameters=True)`
`pg = [p for p in model.parameters() if p.requires_grad]`
`optimizer = AdamW(pg, lr=config['lr'],weight_decay = 0.00001 )`
 `scheduler = get_linear_schedule_with_warmup(
  optimizer, num_warmup_steps=config['num_warmup_steps'], num_training_steps=len(train_loader)*config['epochs'])`

**Comment by dongxuewang-123** [comment]
the loss from 6.23 reduce to 0.48 within 8 epochs,but rebound after 8 epochs
